### ISSUE TITLE
/api/1/vehicles/{id}/data - is it really cached?

### ISSUE THREAD

**Issue by jonahwh** [timdorr/tesla-api#92]
The documentation states that `/api/1/vehicles/{id}/data` is "A cached rollup". Are we sure that it's cached?

I know that it definitely returns up-to-the-moment data when I request it, and I have received ~409~ 408 responses from it when the vehicle is in deep sleep.

**Comment by timdorr** [comment]
A 409 is a Conflict code, so that would indicate a cache is in place (i.e., there is a conflict between the backend state and the cache). It definitely performs better than the individual requests, especially when the vehicle is sleeping.

**Comment by jonahwh** [comment]
Sorry, not 409, 408

**Comment by jonahwh** [comment]
I'll have to test again but I'm all but certain I got the same behavior with this endpoint as with any other state endpoint.

**Comment by jonahwh** [comment]
I just tested this on my 2015 API Model S. I have Energy Saver turned on, and "Always Connected" turned off.

The call to the `wake_up` endpoint said that the vehicle's `state` was `asleep`.

I made repeated calls to `https://owner-api.teslamotors.com/api/1/vehicles/:vehicle_id/data` for 60 seconds, waiting 2 seconds between requests.

Each request came back with a 408 response: `{"response":null,"error":"vehicle unavailable: {:error=\u003e\"vehicle unavailable:\"}","error_description":""}`

After 60 seconds, the vehicle finally woke up and I got a 200 result back with the expected data.
